### ISSUE TITLE
.text-muted does not pass accessibility contrast ratio

### ISSUE THREAD

**Issue by tsathianathan** [twbs/bootstrap#17994]
The .text-muted color does not meet the WCAG 2 contrast ratio against a white background. The color needs to be darker.

**Comment by patrickhlauke** [comment]
in fairness, most color combinations are currently problematic...

**Comment by cvrebert** [comment]
@patrickhlauke Remember to add the "v3" or "v4" tag as appropriate in the future :smile:

**Comment by mdo** [comment]
Closing and making a note of this for #18462.
